README.md

```diff
@@ -559,6 +557,25 @@ esy pesy
 esy build
 ```
 
+### Templates (experimental)
+
+It is possible to create templates to be used by pesy. This is a experimental feature with potential breaking changes without notice.
+
+It works by downloading a git repo and then replacing special strings in filenames and files inside the repo. The special strings are currently these:
+
+| In filename                      | In contents                | Replaced with              |
+| -------------------------------- | -------------------------- | -------------------------- |
+| \_\_PACKAGE_NAME\_\_             | <PACKAGE_NAME>             | package_name_kebab         |
```

Comment:
Should this line and the next really be the same or is this a bug in the template?

Reply:
I think we missed something, originally PACKAGE_NAME_FULL also contained the scope

Reply:
Took a closer look. Indeed, the two variables semantically identical. We can get rid of them. At the time of bootstrapping, packages never have scopes.